backend/tamil-wordle.go

```diff
@@ -76,6 +76,10 @@ func getCurrentWordLenHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func verifyWordHandler(w http.ResponseWriter, r *http.Request) {
+	enableCORS(&w, r)
```

Comment:
Should we pass the `w` as a pointer ? I think it would work if we just pass it normally as it is an interface and is already backed by a pointer. I do not have access to verify this right away. Can you check this once, if you can ?